terasoluna-gfw-functionaltest-parent/pom.xml

```diff
@@ -28,6 +28,27 @@
         <name>TERASOLUNA Framework Team</name>
         <url>http://terasoluna.org</url>
     </organization>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-war-plugin</artifactId>
+                    <configuration>

```

Comment:
`<dependency>`? I received it `<plugin>`.

What you're saying is reasonable.
But for the following reasons, I have added these settings in `parent` pom. Ithink that these are reasonable too.

* `web` should not know directory structure of `env`. It is possible that only the `parent` knows both directory structure of `env` and `web`.
* `parent` manage building whole of  `terasoluna-gfw-functionaltest` projects. So it is reasonable that `parent` manage `<plugingManagement>` for whole of projects.